app/src/main/res/values/styles.xml

```diff
@@ -34,6 +41,11 @@
         <item name="android:textSize">24sp</item>
     </style>
 
+    <style name="Text.Big.Light">
```

Comment:
I think this one is more related to Text.Heading instead of Text.Big . So maybe Text.Heading.Dark as it's black :)